packages/core/src/app/customer/StripeGuestForm.tsx

```diff
@@ -169,11 +169,8 @@ const StripeGuestForm: FunctionComponent<StripeGuestFormProps & FormikProps<Gues
 
     return (
         <>
-            <div className="checkout-form">
-                <LoadingOverlay
-                    hideContentWhenLoading
-                    isLoading={ isStripeLoading }
-                >
+            <AddressFormSkeleton isLoading={isStripeLoading}/>
```

Comment:
Customer step has its own loading skeleton, can we use this one?
`import { CustomerSkeleton } from '@bigcommerce/checkout/ui';`

Reply:
I had no idea about CustomerSkeleton, but it makes sense, thank you very much @bc-peng

Reply:
<img width="713" alt="image" src="https://user-images.githubusercontent.com/42154828/217656938-ae52427d-fef4-4867-8428-28510152543d.png">